gallery/src/pageConfig.ts

```diff
@@ -53,57 +53,57 @@ import NxStatefulCheckboxPage from './components/NxStatefulCheckbox/NxStatefulCh
 
 const pageConfig: PageConfig = {
   'Styles - HTML Elements': {
-    'nx-page-title': NxPageTitlePage,
-    'nx-tile': NxTilePage,
-    'nx-btn': NxBtnPage,
-    'nx-list': NxListPage,
-    'nx-text-input': NxTextInputStylesPage,
     'nx-alert': NxAlertPage,
-    'nx-table': NxTableStylePage,
-    'nx-grid': NxGridPage,
+    'nx-btn': NxBtnPage,
     'nx-counter': NxCounterPage,
+    'nx-grid': NxGridPage,
     'nx-icon': NxIconPage,
+    'nx-list': NxListPage,
+    'nx-page-title': NxPageTitlePage,
+    'nx-table': NxTableStylePage,
+    'nx-text-input': NxTextInputStylesPage,
     'nx-threat-number': NxThreatNumberPage,
-    'Form Layout Styles': NxFormLayoutPage
+    'nx-tile': NxTilePage
   },
   'Styles - Mixins': {
     'nx-container-helpers': NxContainerHelpersPage
   },
+  'Layout Examples': {
+    'Form Layout Styles': NxFormLayoutPage
+  },
   'Guidelines': {
     'Form Validation Guidelines': FormValidationPage
   },
   'React Components': {
+    NxAlert: NxAlertComponentsPage,
+    NxBackButton: NxBackButtonPage,
     NxButton: NxButtonPage,
     NxCheckbox: NxCheckboxPage,
-    NxRadio: NxRadioPage,
-    NxTextInput: NxTextInputPage,
-    NxBackButton: NxBackButtonPage,
-    NxLoadingSpinner: NxLoadingSpinnerPage,
+    NxStatefulCheckbox: NxStatefulCheckboxPage,
+    NxDropdown: NxDropdownPage,
+    NxStatefulDropdown: NxStatefulDropdownPage,
+    NxFilterInput: NxFilterInputPage,
+    NxFontAwesomeIcon: NxFontAwesomeIconPage,
     NxLoadError: NxLoadErrorPage,
     NxLoadWrapper: NxLoadWrapperPage,
+    NxLoadingSpinner: NxLoadingSpinnerPage,
     NxModal: NxModalPage,
-    NxAlert: NxAlertComponentsPage,
-    NxThreatBar: NxThreatBarPage,
-    NxFontAwesomeIcon: NxFontAwesomeIconPage,
-    NxVulnerabilityDetails: NxVulnerabilityDetailsPage,
+    NxPolicyThreatSlider: NxPolicyThreatSliderPage,
+    NxRadio: NxRadioPage,
     NxSubmitMask: NxSubmitMaskPage,
+    NxStatefulSubmitMask: NxStatefulSubmitMaskPage,
     NxTable: NxTablePage,
-    NxTreeView: NxTreeViewPage,
+    NxTextInput: NxTextInputPage,
+    NxStatefulTextInput: NxStatefulTextInputPage,
+    NxThreatBar: NxThreatBarPage,
     NxTooltip: NxTooltipPage,
-    NxTreeViewRadioSelect: NxTreeViewRadioSelectPage,
+    NxTreeView: NxTreeViewPage,
     NxTreeViewMultiSelect: NxTreeViewMultiSelect,
-    NxPolicyThreatSlider: NxPolicyThreatSliderPage,
-    NxFilterInput: NxFilterInputPage,
-    NxDropdown: NxDropdownPage
-  },
-  'Stateful React Components': {
-    NxStatefulCheckbox: NxStatefulCheckboxPage,
-    NxStatefulTextInput: NxStatefulTextInputPage,
-    NxStatefulSubmitMask: NxStatefulSubmitMaskPage,
-    NxStatefulDropdown: NxStatefulDropdownPage,
     NxStatefulTreeViewMultiSelect: NxStatefulTreeViewMultiSelect,
-    NxStatefulTreeViewRadioSelect: NxStatefulTreeViewRadioSelectPage
-  }
+    NxTreeViewRadioSelect: NxTreeViewRadioSelectPage,
```

Comment:
The only downside to this is I was able to simply sort the components alphabetically in my text editor, but with the stateful components next to their non-stateful versions we just have to make sure we stick to the correct ordering in the future. It's probably fine now that we've done the ordering once.

Reply:
> The only downside to this is I was able to simply sort the components alphabetically in my text editor, but with the stateful components next to their non-stateful versions we just have to make sure we stick to the correct ordering in the future. It's probably fine now that we've done the ordering once.

And we have to remember when we add new stateful components to place them next to their non-stateful equivalents, but I think the grouping is worth it.

Reply:
> The only downside to this is I was able to simply sort the components alphabetically in my text editor, but with the stateful components next to their non-stateful versions we just have to make sure we stick to the correct ordering in the future. It's probably fine now that we've done the ordering once.

When we discussed this change earlier didn't we decide to rename Stateful components to put "Stateful" at the end?

Reply:
I think that'd be less confusing conceptually, I may change my mind if I saw it ;)